test/playwright/utils/navigation.ts

```diff
@@ -464,19 +464,12 @@ export const enableNewHeader = async (page: Page) => {
 
 export const enableOldHeader = async (page: Page) => {
   // Add the new_header cookie
-  await page.context().addCookies([
-    {
-      name: "features",
-      value: "%7B%22new_header%22%3A%22off%22%7D",
-      domain: "localhost",
-      path: "/",
-    },
-  ])
+  await setCookies(page.context(), { features: { new_header: "off" } })
```

Comment:
This way of setting the `new_header` flag is so much more cleaner, compared to the old approach of Playwright visiting the `/preferences/page` and toggling the checkbox!

The use of `setCookies` should be extended to the `enableNewHeader` function just above it.